evm/src/cpu/columns/general.rs

```diff
@@ -71,18 +93,39 @@ impl<T: Copy> BorrowMut<[T; NUM_SHARED_COLUMNS]> for CpuGeneralColumnsView<T> {
     }
 }
 
-#[derive(Copy, Clone)]
-pub(crate) struct CpuKeccakView<T: Copy> {
-    pub(crate) input_limbs: [T; 50],
-    pub(crate) output_limbs: [T; 50],
-}
-
 #[derive(Copy, Clone)]
 pub(crate) struct CpuArithmeticView<T: Copy> {
     // TODO: Add "looking" columns for the arithmetic CTL.
     tmp: T, // temporary, to suppress errors
 }
 
+#[derive(Copy, Clone)]
+pub(crate) struct CpuJumpsView<T: Copy> {
+    // Assuming a limb size of 32 bits.
+    pub(crate) input0: [T; 8],
```

Comment:
If I understand correctly, these arrays are like copies of `mem_value[0]` etc., and will later be constrained to match. What do you think of placing constraints on `mem_value` directly?

In this case we have columns to spare anyway, but in e.g. a logic row, it seems like we'd want to avoid any redundancy since we wouldn't have columns to spare.

Reply:
We chatted and decided to do this in a future change.